examples/gitlab-centos7-atomicapp/README.md

```diff
@@ -0,0 +1,157 @@
+This is an atomic application based on the nulecule specification. Docker and Kubernetes are the supported providers. You'll need to run this from a workstation that has the atomic command and kubectl client that can connect to a kubernetes master.
+
+It is 3 a tier application based on redis, postgresql and gitlab. This is also an example Nulecule having multiple tier artifacts rather using graphs.
+
+### Option 1: Interactive
+
+Run the image. It will automatically use kubernetes as the orchestration provider.  It will prompt for all parameters in the Nulecule file that do not have default values:
+
+    $ [sudo] atomic run projectatomic/gitlab-centos7-atomicapp
+
+## Option 2: Unattended
```

Comment:
Right, Option 2 sounds like 'configure deployment' rather than unattended, by creating an answers.conf file (and I would get rid of the opportunity to copy and paste from github... ) the user configures the deployment (run) he is going to do.